netty/src/test/java/io/grpc/transport/netty/NettyClientHandlerTest.java

```diff
@@ -176,42 +206,42 @@ public void cancelShouldSucceed() throws Exception {
   }
 
   @Test
-  public void cancelBeforeStreamAssignedShouldSucceed() throws Exception {
+  public void cancelWhileBufferedShouldSucceed() throws Exception {
     handler.connection().local().maxStreams(0);
     handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream), promise);
     mockContext();
-    verify(stream, never()).id(any(Integer.class));
```

Comment:
This was verifying that the stream was indeed buffered. Now that the id is set even if it is buffered, checking stream.id() doesn't add any value for this test.

If we had a fake channel instead of mock, we could check that the promise hasn't completed... but alas.